e2e/cypress/integration/messaging/copy_post_text_spec.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
FYI @jgilliam17 I wasn't able to actually fully run this test because it looks like the E2E tests for the post menu are broken because of the new actions menu stuff having a tutorial tip - not sure if there's a way around that

Reply:
I'll check this test after merge (after action menu fix is also merged) and create a follow up if we need to fix.